Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -1061,7 +1061,8 @@
 
 #if !defined(ENABLE_WRITING_TOOLS) \
     && ((PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 150000) \
-    || (((PLATFORM(IOS) && !TARGET_OS_VISION) || PLATFORM(MACCATALYST)) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 180000)) \
+    || ((PLATFORM(IOS) || PLATFORM(MACCATALYST)) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 180000) \
+    || (PLATFORM(VISION) && __VISION_OS_VERSION_MAX_ALLOWED >= 20400)) \
```

Comment:
I'm a bit curious about why this is an SDK version check now (`MAX_ALLOWED`), rather than target OS version check (`MIN_REQUIRED`). Shouldn't make a difference in practice, though.

Reply:
I did it because I thought that MAX_ALLOWED is now always preferred over MIN_REQUIRED. I can change it back though if you want?